File: EducationManagement.Database/Models/SubjectLevel.cs
using System;
using System.Collections.Generic;

namespace EducationManagement.Database.Models
{
    public partial class SubjectLevel
    {
        public SubjectLevel()
        {
            Group = new HashSet<Group>();
            Unit = new HashSet<Unit>();
        }

        public int SubjectId { get; set; }
        public int LevelId { get; set; }
        public int? EntryTaskId { get; set; }
        public int Price { get; set; }

        public Task EntryTask { get; set; }
        public Level Level { get; set; }
        public Subject Subject { get; set; }
        public ICollection<Group> Group { get; set; }
        public ICollection<Unit> Unit { get; set; }
        public ICollection<SubjectLevelFile> SubjectLevelFiles { get; set; }
    }
}

File: EducationSystem.WEB/src/app/_models/SubjectLevel.ts
export class SubjectLevel {
    public SubjectId: number;
    public LevelId: number;
    public SubjectName: string;
    public LevelName: string;
    public EntryTaskId: number;
    public MinEnrtyTaskScore: number;
    public Price: number;

    constructor(levelName?: string, levelId?: number, subjectName?: string, subjectId?: number,
        entryTaskId?:number, price?: number, minEntryScore?: number) {
        this.LevelName = levelName;
        this.LevelId = levelId;
        this.EntryTaskId = entryTaskId;
        this.Price = price;
        this.SubjectName = subjectName;
        this.SubjectId = subjectId;
        this.MinEnrtyTaskScore = minEntryScore;
    }

    public get Name(): string {
        return this.SubjectName + " " + this.LevelName;
    }
}